integration_test/external_tests/commons/test_util.dart

```diff
@@ -62,3 +62,15 @@ Future<void> enterSeeds(AppiumWebDriver driver, String seeds) async {
   var confirmButton = await driver.findElement(confirmButtonLocator);
   await confirmButton.click();
 }
+
+Future<void> scroll(driver, scrollUIAutomator) async {
+  var finder = await AppiumBy.uiautomator(scrollUIAutomator);
+  await driver.findElement(finder);
+}
+
+Future<void> scrollSettingPage(driver) async {
```

Comment:
@ppupha Please update this function to scroll to each element you want to scroll. We will have 2 params, 1 is the driver, 1 is the Appium element we want to scroll to it.

Reply:
Updated. The 1 param is driver, the 2 is (a part of) content-decs of element we want to scroll to.